src/libsyncengine/jobs/network/API_v2/movejob.h

```diff
@@ -28,14 +28,14 @@ class MoveJob : public AbstractTokenNetworkJob {
     public:
         MoveJob(const std::shared_ptr<Vfs> &vfs, int driveDbId, const SyncPath &destFilepath, const NodeId &fileId,
                 const NodeId &destDirId, const SyncName &name = Str(""));
-        ~MoveJob();
+        ~MoveJob() final;
```

Comment:
Making a constructor or destructor `final` means that the class cannot be derived from anymore (which can be problematic for mocking in tests). Is there a specific reason for making this destructor `final`?
```suggestion
        ~MoveJob() override;
```

Reply:
No, I agree, `override` is better here